File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/InvalidUserRevocIndexException.java
package org.hyperledger.indy.sdk;

public class InvalidUserRevocIndexException extends IndyException
{
	private final static String message = "The user revocation registry index specified is invalid.";

	InvalidUserRevocIndexException(int sdkErrorCode) 
    {
    	super(message, sdkErrorCode);
    }
}

File: wrappers/dotnet/indy-sdk-dotnet/AnonCredsApi/InvalidUserRevocIndexException.cs
namespace Hyperledger.Indy.AnonCredsApi
{
    /// <summary>
    /// Exception thrown when an invalid user revocation registry index is used.
    /// </summary>
    public class InvalidUserRevocIndexException : IndyException
    {
        const string message = "The user revocation registry index specified is invalid.";

        /// <summary>
        /// Initializes a new InvalidUserRevocIndexException.
        /// </summary>
        internal InvalidUserRevocIndexException() : base(message, (int)ErrorCode.AnoncredsInvalidUserRevocIndex)
        {

        }
    }

}
